crates/iota-replay/src/tests.rs

```diff
@@ -30,7 +29,8 @@ async fn verify_latest_tx_replay_impl() {
     let urls: Vec<_> = default_cfg
         .base_network_configs
         .iter()
-        .filter(|q| q.name != "devnet") // Devnet is not always stable
+        // TODO: enable this when mainnet is launched
```

Comment:
Could you create an issue for this TODO?

Reply:
Sure

Reply:
https://github.com/iotaledger/iota/issues/4405

Reply:
Thanks 🙏🏻